README.md

```diff
@@ -1,144 +1,211 @@
+| --------- | ----------- | ----- |
+| assets | Files to be used in running Terminus | |
+| bin | User-executable scripts | |
+| config | Files to be used in configuring Terminus | |
+| docs | Documentation about the source code and sample scripts | |
+| scripts | Scripts for maintaining the source code | |
+| src | Source code of the application | |
+| templates | Files used for templating the documentation | |
+| tests | Tests for maintaining the codebase | |
+| utils | Utility scripts used by components of Terminus | |
+ 
+## Developing & Contributing
+1. See the [CONTRIBUTING](CONTRIBUTING.md) document.
+2. Create an issue on this repository to discuss the change you propose should be made.
+3. Fork this repository.
+4. Clone the forked repository.
+5. Run `composer install` at the repository root directory to install all necessary dependencies.
+6. Make changes to the code.
+7. Run the test suite. The tests must pass before any code will be accepted.
+8. Commit your changes and push them up to your fork.
+9. Open a pull request on this repository to merge your fork.
+
+Your pull request will be reviewed by Pantheon and changes may be requested before the code is accepted.
+
+## Testing
+Tests are run via the `.scripts/test.sh` script. Components thereof can be run as follows:
+
+- `composer cs` runs the code sniffer to ensure all code is appropriately formatted.
+- `composer phpunit` runs the PHPUnit unit tests.
+- `composer behat` runs the Behat feature tests
+ 
+## Support
+Please make ready the steps to reproduce the issue, outputs, pertinent information about your system, and what you
+believe the correct reaction of the system ought to be. Reporting of issues encountered should happen in one of two
+ways:
+
+### Information that helps us help you
+If we cannot duplicate an issue, we cannot resolve it. Giving us as much information about the problem you're having
+as you can will help reduce the amount of time between an issue's being reported and being resolved. This is typically
+the most helpful information:
+
+- The result of running `terminus self:info`.
+- The output of the issue in debug mode. (Run the command with `--vvv` appended to it.)
+
+### If yours is a problem with Terminus itself:
+1. Search [Terminus' issues on GitHub](https://github.com/pantheon-systems/terminus/issues) to see whether another
+user has reported the same issue you are experiencing.
+2. If the problem you are experiencing is not in the issues, you can open a new issue. Please include the helpful
+information you have gathered.
+3. If you find your problem in an issue, feel free to add your issue information in the comments and/or subscribe to
+the issue's notifications.
+
+### If the problem is with the Pantheon platform
+Head over to [your support tickets in the Pantheon Dashboard](https://dashboard.pantheon.io/users/#support) and
+submit a new issue ticket. Please include the helpful information you have gathered.
+ 
+## Performance and Benchmarking
```

Comment:
Not sure this section is necessary for the terminus client? Esp if we're not measuring it and holding contributions to any baseline.